darts/dataprocessing/pipeline.py

```diff
@@ -14,8 +14,10 @@
 
 class Pipeline:
     def __init__(self,
-                 transformers: Sequence[BaseDataTransformer[TimeSeries]],
-                 copy: bool = False):
+                 transformers: Sequence[BaseDataTransformer],
+                 copy: bool = False,
+                 verbose: bool = None,
```

Comment:
This should probably default to `False`. Next line should probably default to the default of `transformer.n_jobs`.

Reply:
Mb, I read the comment and it makes sense to have a `None` default.